plugin.video.iranintl/addon.xml

```diff
@@ -19,6 +18,6 @@
     <forum></forum>
```

Comment:
Please remove the empty `<forum>` tag the next time.

Reply:
Thanks, will do!